### ISSUE TITLE
Warning : Failed to locate origin index for [Origin Type]

### ISSUE THREAD

**Issue by broxen** [Segs/Segs#159]
### Issue Summary
Upon creating a character the server window outputs the following error message: `Warning : Failed to locate origin index for "Technology"` where `Technology` is whatever Origin you've chosen.

### System Info
Latest from Git 02ec61b
Qt 5.9.2 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 7.2.1 20170915 (Red Hat 7.2.1-2)) on "xcb" 
OS: Fedora 26 (Workstation Edition) [linux version 4.13.8-200.fc26.x86_64]

**Comment by nemerle** [comment]
Question is, is this a failure when searching for the "Technology" string ?, or is it a problem with loading origin/class data files ?
Does the server report a correct load of:
* origins.bin
* villain_origins.bin
* classes.bin
* villain_classes.bin

**Comment by broxen** [comment]
Yes, it appears to be loading the bins correctly. 

Also, it only shows this error when creating a _new_ character, not when loading an existing character from the dbase.

My console at runtime below.
```
Super Entity Game Server
http://segs.sf.net/
Copyright (c) 2006-2017 Super Entity Game Server Team (see Authors.txt)
This software is licensed! (See License.txt for details)

segs v0.4.0 alpha (The Real Beginning)
main
Debug   : Config file is missing 'location_addr' entry, will try to use default
Debug   : Config file is missing 'auth_pass' entry, will try to use default
Warning : Reading map data from ./data/bin/ folder
Debug   : Reading ./data/bin/costume.bin ... OK
Debug   : Reading ./data/bin/supergroupColors.bin ... OK
Debug   : Loading origins:
Debug   : Reading ./data/bin/origins.bin ... OK
Debug   : Reading ./data/bin/villain_origins.bin ... OK
Debug   : Loading classes:
Debug   : Reading ./data/bin/classes.bin ... OK
Debug   : Reading ./data/bin/villain_classes.bin ... OK
Warning :  All map data read
Warning :  Postprocessing runtime data ..  Hashes filled
```

**Comment by broxen** [comment]
Fixed by latest commit #161
